_episodes/01-introduction.md

```diff
@@ -32,23 +32,35 @@ keypoints:
+The image below shows some differences between artificial intelligence, Machine Learning and Deep Learning. 
+
+![An infographics showing the relation of AI, ML, NN and DL](../fig/01_AI_ML_DL_differences.svg){: width="500px" }
+The image above is by Tukijaaliwa, CC BY-SA 4.0, via Wikimedia Commons, [original source]( https://en.wikipedia.org/wiki/File:AI-ML-DL.svg)
 
 
 #### Neural Networks
 
-A neural network is an artificial intelligence technique loosely based on the way neurons in the brain work. A neural network consists of connected computational units called neurons. Each neuron takes the sum of all its inputs, performs some, typically non-linear, calculation on them and produces one output. This calculation is called the activation function. The connections between neurons are called edges, these edges typically have a weight associated with them. This weight determines the 'strength' of the connection, these weights are adjusted during training. In this way, the combination of neurons and edges describe a computational graph, an example can be seen in the image below. In most neural networks neurons are aggregated into layers. Signals travel from the input layer to the output layer, possibly through one or more intermediate layers called hidden layers.
+A neural network is an artificial intelligence technique loosely based on the way neurons in the brain work. A neural network consists of connected computational units called neurons. Each neuron has one or more inputs and takes the sum of these each multiple by a weight, performs some, typically non-linear, calculation on the sum and produces one output. This calculation is called the activation function. An extra input called a bias term is usually attached with a value fixed at 1.
 
-Neural networks aren't a new technique, they have been around since the late 1940s. But until around 2010 neural networks tended to be quite small, consisting of only 10s or perhaps 100s of neurons. This limited them to only solving quite basic problems. Around 2010 improvements in computing power and the algorithms for training the networks made much larger and more powerful networks practical. These are known as deep neural networks or Deep Learning.
+![A diagram of a single artificial neuron](../fig/01_neuron.png){: width="600px"}
 
-![An infographics showing the relation of AI, ML, NN and DL](../fig/AI_ML_DL_bubble_square_draft.png){: width="600px" }
+Multiple neurons can be joined together by connecting the output of one to the input of another. These connections are called edges. The weights determines the 'strength' of the connection, the weights are adjusted during training. In this way, the combination of neurons and edges describe a computational graph, an example can be seen in the image below. In most neural networks neurons are aggregated into layers. Signals travel from the input layer to the output layer, possibly through one or more intermediate layers called hidden layers. 
```

Comment:
This paragraph mentions the term `edge` multiple times. Will this be reused later? If not, I'd contemplate to not stress it too much to keep the cognitive load low.